package.json

```diff
@@ -42,8 +41,8 @@
     "url": "https://github.com/facebook/jest"
   },
   "scripts": {
-    "prepublish": "jshint --config=.jshintrc --exclude=src/coverageTemplate.js src && node bin/jest.js",
-    "test": "jshint --config=.jshintrc --exclude=src/coverageTemplate.js src && node bin/jest.js"
+    "prepublish": "eslint src && node bin/jest.js",
```

Comment:
2 things.

1. Let's make a separate `lint` script here, then use that.
2. Let's make that lint script just `eslint .` and add the other directories we don't want linted to `.eslintignore`